src/elexmodel/models/BootstrapElectionModel.py

```diff
@@ -1024,6 +1024,7 @@ def compute_bootstrap_errors(
         self.weighted_z_test_pred = z_test_pred * weights_test
         self.ran_bootstrap = True
         self.n_contests = aggregate_indicator.shape[1]
+        # import IPython; IPython.embed()
```

Comment:
Can you remove this?  (Also I had never heard of `IPython.embed()` before and it's cool 😄 )

Reply:
Lol yes absolutely

Reply:
done